src/components/calculator.js

```diff
@@ -4,26 +4,26 @@ const buttonsGroup = require("./buttons-group");
 function calculator(id) {
 	const buttonsTop = buttonsGroup("buttons-top", [
 		button("C","clear-all upper left"),
-		button("%","bottom"),
+		button("+/-","less-more upper"),
 		button("CE","del upper")
 	]);
 	const buttonsMiddle = buttonsGroup("buttons-middle",[
-		button("1"), button("2"),button("3"),
-		button("4"), button("5"),button("6"),
-		button("7"), button("8"),button("9"),
+		button("1","left"), button("2"),button("3"),
```

Comment:
why do you need to put a left class for those buttons?

Reply:
yea true,i think i can do the same thing just with the div.
i'm using this class just to have a margin around the buttons, but now it's kinda useless